core/experiments/wp-cli.php

```diff
@@ -165,13 +196,15 @@ private function get_plural( $experiments ) {
 	 * @return bool true when all experiments exist, otherwise false
 	 */
 	private function check_experiments_exist( $experiments_manager, $experiments ) {
+        $allExperimentsExist = true;
 		foreach ( $experiments as $experiment ) {
 			$feature = $experiments_manager->get_features( $experiment );
 			if ( ! $feature ) {
-				return false;
+                \WP_CLI::error( "Experiment '$experiment' do not exist" );
```

Comment:
Indendation seems off.

Reply:
@hein-obox  Looks ok to me in the editor. Can you pull this branch to confirm?

Reply:
![image](https://github.com/elementor/elementor/assets/130742176/e6987fd3-defb-4b0f-af31-92c72f9a5e49)

Reply:
@Barcov I think it is most important that it looks correct on github: https://github.com/elementor/elementor/blob/0bc743641f49f561cb494ae49fa27e084b4a1472/tests/playwright/utilities/terminal.js. 

Currently it seems off.